servicedirectory/quickstart_test.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import environ
+import uuid
+
+import pytest
+import quickstart
+
+from google.api_core import exceptions
+from google.cloud import servicedirectory_v1beta1
+
+PROJECT_ID = environ['GOOGLE_CLOUD_PROJECT']
+LOCATION_ID = 'us-east1'
+NAMESPACE_ID = f'test-namespace-{uuid.uuid4().hex}'
+
+
+@pytest.fixture(scope='module')
+def client():
+  return servicedirectory_v1beta1.RegistrationServiceClient()
+
+
+@pytest.fixture(scope='module')
+def namespace(client):
+  return servicedirectory_v1beta1.Namespace(
```

Comment:
Suggestion:

Can we make the fixture `namespace` to create the namespace (and delete it after the test), so that we don't need to have `list_namespace` fixture any more?

In the test, we can call list_namespaces and check the output.

Reply:
ah, i think i misunderstood what you mean the first time. gotcha.